cmd/lifecycle/rebaser.go

```diff
@@ -148,3 +156,7 @@ func (r *rebaseCmd) Exec() error {
 	}
 	return nil
 }
+
+func (r *rebaseCmd) resolveKeychain() (authn.Keychain, error) {
+	return auth.ResolveKeychain(cmd.EnvRegistryAuth, r.imageNames)
```

Comment:
This needs to include `r.runImageRef`.

Reply:
This is another case where we will need to ensure we have resolved the best run image first.